src/Sulu/Bundle/AdminBundle/Resources/js/containers/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from 'react';
+import {observer} from 'mobx-react';
+import classNames from 'classnames';
+import sidebarStore from './stores/SidebarStore';
+import sidebarViewRegistry from './registries/SidebarViewRegistry';
+import sidebarStyles from './sidebar.scss';
+
+type Props = {
+    className?: string,
+};
+
+@observer
+export default class Sidebar extends React.Component<Props> {
+    render() {
+        if (!sidebarStore.view) {
+            return null;
+        }
+
+        const Component = sidebarViewRegistry.get(sidebarStore.view);
+        const {
+            className,
+        } = this.props;
+
+        const sidebarClass = classNames(
+            sidebarStyles.sidebar,
+            className
+        );
+
+        return (
+            <aside className={sidebarClass}>
```

Comment:
If we rebuild the `Application` as mentioned above I would suggest to make this a `div` and wrap it in an `aside` tag in the `Application`.

Reply:
After seeing the rest of the code I would leave that as it is, but add a `return null` to disable the sidebar when nothing is in the `Sidebar`

Reply:
there is a `return null` when view is null